lms/djangoapps/certificates/management/commands/cert_whitelist.py

```diff
@@ -57,6 +57,18 @@ def handle(self, *args, **options):
         if not course_id:
             raise CommandError("You must specify a course-id")
 
+        def update_user_whitelist(username, add=True):
```

Comment:
@e0d , actually, there is an other function like `get_user_from_identifier`: https://github.com/edx/edx-platform/blob/master/lms/djangoapps/instructor/views/tools.py#L84-L97

Maybe we could pull that out into a more general util? Or extend the `User` class and give it this as a class method?

... Or just keep it like this and not get too fancy :)